app/clinic/controllers/patientSummaryController.js

```diff
@@ -107,8 +107,8 @@
     function initICD10Diagnosis() {
       var concept = "e1eb7806-1d5f-11e0-b929-000c29ad1d07";//TODO: create in configuration file
 
-      return observationsService.getObs(patientUuid, concept).success (function (data) {
-        var filtered = commonService.filterRetired(data.results);//TODO: filter must be dome in backend system
+      return observationsService.getObs(patientUuid, concept).then (function (obs) {
```

Comment:
Because ```success``` is deprecated in most recent versions of angular and it goes with the [native promise api](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).
I Think using success and error jus causes confusion.